src/utils.jl

```diff
@@ -48,3 +48,27 @@ function standardize_predictors(X::AbstractMatrix)
     end
     return μ_X, σ_X, X_std
 end
+
+"""
+    tuple_length(::NTuple{N, Any}) where {N} = Int(N)
+
+This is a hack to get the length of any tuple.
+"""
+tuple_length(::NTuple{N,Any}) where {N} = Int(N)
+
+"""
+    convert_str_to_indices(v::AbstractVector)
+
+Converts a vector `v` to a vector of indices, i.e. a vector where all the entries are
+integers. Returns a tuple with the first element as the converted vector and the
+second element a `Dict` specifying which string is which integer.
+"""
```

Comment:
And why is this method useful? Can you add an example of a valid `v` and it's output?

Reply:
This is useful for Varying-Intercept models (See https://storopoli.io/Bayesian-Julia/pages/10_multilevel_models/):
```julia
@model function varying_intercept(X, idx, y; n_gr=length(unique(idx)))
....
end
```

Normally `v` would be a vector of groupmembership such as "group_1", "group_2" etc. I need to convert it to a vector of Int's for the underlying Turing model (if we are doing varying-intercepts).

Reply:
Clear. Can you add it to the code? :)

Reply:
To the docstring, I mean

Reply:
done in 9f381d3.